### DISCUSSION TITLE
Open page with all Bard sets closed?

### DISCUSSION THREAD

**Discussion by Sennik** [statamic/cms#4638] [Troubleshooting] [answered]
When I open a page where I use Bard with multiple texts and sets it quickly becomes cumbersome to scroll to all the visual data to get to the set I want to change the content of. 

It would be nice when all sets are (configerable) initially closed when loading a page. Or maybe better, they all remember the state from the previous time the page with the bard field was saved. Open all closed will be fine though.

Is this very difficult to implement?

**Comment by edalzell** [comment]
Try putting `collapse: true` into the blueprint for that bard field.

**Reply by Sennik** [reply]
Thanks, good suggestion :-) I do not get it to work though:

```
  field:
    always_show_set_button: true,
    collapse: true
    sets:
    ...
```
Wrong place?

**Reply by jasonvarga** [reply]
That's not a thing 😄

**Comment by jasonvarga** [comment, marked as answer]
It's not possible to start with them closed, but you can click the collapse sets button to close them all at once.

![image](https://user-images.githubusercontent.com/105211/140165378-0d8b3679-9734-4a82-9d88-22a78b8c8842.png)

This feature request already exists at https://github.com/statamic/ideas/issues/407 though.

**Reply by Sennik** [reply]
I know and use the button. But after 10 times opening the page with the bard field I strongly desire an option to have them collapsed after opening. So it still would be nice to have :-)

**Reply by duncanmcclean** [reply]
If you've not already, go and add a 👍  reaction to the feature request: statamic/ideas#407
